lib/eblox/data/post.ex

```diff
@@ -27,7 +27,8 @@ defmodule Eblox.Data.Post do
 
     @impl Md.Transforms
     def apply(md, text) do
-      {:a, %{class: "tag", href: URI.encode_www_form(@href <> text)}, [md <> text]}
+      href = @href <> String.downcase(URI.encode_www_form(text))
+      {:a, %{class: "tag", href: href}, [md <> text]}
```

Comment:
Nah, I finally got the root of all the misunderstanding :) We need to preserve the downcased tag as `data-` attribute to avoid inconsistency.
```suggestion
      href = @href <> URI.encode_www_form(text)
      tag = String.downcase(text)
      {:a, %{class: "tag", "data-tag": tag, href: href}, [md <> text]}
```
See below.